### DISCUSSION TITLE
[Question][Hw0] Can`t connect to the docker container with vscode

### DISCUSSION THREAD

**Discussion by Jihwan-Yoon** [prosyslab-classroom/cs348-information-security#646] [Q&A]
Name: Jihwan Yoon

![image](https://user-images.githubusercontent.com/78306242/222682989-c6949b2d-a2af-4302-bd4b-67e1c0df8d5b.png)

![image](https://user-images.githubusercontent.com/78306242/222683084-a0cef18c-c924-428c-9df0-0e63691536c3.png)

I succeeded in connecting the container yesterday.

But it doesn`t work now.

I don`t know why the vscode can not connect to the container.

Thank you!!

**Comment by tncks0121** [comment]
See the log by clicking "Starting Dev Container (show log)", it was helpful for me in most cases

**Comment by Jihwan-Yoon** [comment]
![image](https://user-images.githubusercontent.com/78306242/222686697-53492271-febc-4bf0-bc32-31b4e7c8c04e.png)

I have tried it but couldn`t understand it...

**Comment by m-spitfire** [comment]
@Jihwan-Yoon I suggest you to use the provided KCloud machines as they're simpler to set up and develop on rather than Docker, I feel like. See #202.

I think because you're using Windows, the docker engine is slow, because it uses a virtual machine(WSL 2) under the hood, but I am not too sure.

**Comment by Jihwan-Yoon** [comment]
Oh, I see.

I tried Docker first, because I couldn`t find klms IP notice yesterday.

KCloud works well!!

Thank you
